esphome/components/lg_uart/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""
+Simple UART based component to set/poll various settings for LG screens.
+LG provides generic documentation that describes many different things that can be set.
+Not all models/firmware-versions support setting all things.
+
+Additionally, not all models support polling the state of various things.
+Each of the different things that can be controlled can be best represented with a switch (power/mute) or
+    sensor (volume level?) or slider (volume) ... etc
+"""
+
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import uart
+from esphome.const import CONF_UART_ID, CONF_ID
+
+CODEOWNERS = ["@kquinsland"]
+DEPENDENCIES = ["uart"]
+MULTI_CONF = False
+
+# Hub needs the ID of the UART and set ID
+CONF_SCREEN_NUMBER = "screen_number"
+
+# Clients
+CONF_LG_UART_ID = "lg_uart_id"
+CONF_LG_UART_CMD = "lg_uart_cmd"
+
+CONF_DECODE_BASE = "decode_base"
+CONF_DECODE_BASE_TYPES = [10, 16]
```

Comment:
The `CONF_` prefix is for the config strings, not for values
```suggestion
DECODE_BASE_TYPES = [10, 16]
```